SampleBrowser/SampleBrowser/SceneDelegate.swift

```diff
@@ -16,9 +17,15 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
                willConnectTo session: UISceneSession,
                options connectionOptions: UIScene.ConnectionOptions) {
         guard let windowScene = (scene as? UIWindowScene) else { return }
-        let baseViewController = RootViewController(engineProvider: engineProvider)
+        let windowUUID = UUID()
+        let baseViewController = RootViewController(engineProvider: engineProvider, windowUUID: windowUUID)
         window = UIWindow(windowScene: windowScene)
```

Comment:
I just took a quick look at this so I might have missed something, but offhand I'm a bit confused by our theme-related code. Is it working correctly during testing? Because I can't see any place where the UI here is referencing this window UUID that is being generated, in order to pass it along to the ThemeManager for the `currentTheme()`. 🤔

Reply:
@mattreaganmozilla `RootViewController` is using the windowUUID in `applyTheme()`. It was added in [this PR](https://github.com/mozilla-mobile/firefox-ios/pull/19684).

Reply:
> And yes, it's mostly working - there are some cases where you have to close and reopen the app though.

Ok LMK if you continue to have issues there and I can take a look (if you want any help fixing it)

Reply:
Will do. I'll create tasks when I see the issues pop up.